### PR TITLE
Added Vietnamese translation

### PR DESCRIPTION
**Pull request by Dummynosrev**
[BASE/SETUP/REACTOS/LANG]
Took waay too long to figure stuff out.
If I'm doing anything incorrectly, tell me. 
- Added Vietnamese translation

### CODE REVIEW

File: base/setup/reactos/lang/vi-VN,rc [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    IDS_NO_TXTSETUP_SIF "Không thể tìm được tập tin 'txtsetup.sif'\ntrình Thiết lập không thể tiếp tục"
+    IDS_CAPTION "Cài đặt ReactOS"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_PARTITION_NAME "Tên"
+    IDS_PARTITION_SIZE "Kích cỡ"
+    IDS_PARTITION_TYPE "Loại định dạng"
+ENDcaoch\u1ec9
```
Comment (binarymaster) [inline base/setup/reactos/lang/vi-VN,rc]: Something wrong after END.

File: base/setup/reactos/lang/vi-VN,rc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    IDS_DEVICESUBTITLE "Chọn những cài đặt cho màn hình và bàn phím."
+    IDS_DRIVETITLE "Thiết lập phần ổ cứng để cài đặt và thư mục hệ thống"
+    IDS_DRIVESUBTITLE "Chuẩn bị phần ổ cứng, thư mục hệ thống và boot loader."
+    IDS_PROCESSTITLE "Thiết lập phần ổ cứng, sao chép tập tin và thiết lập hệ thống"
+    IDS_PROCESSSUBTITLE "Tạo và định dạng phần ổ đĩa, sao chép thư  mục, cài đặt và thiết lập boot loader"
+    IDS_RESTARTTITLE "Giai đoạn đầu của việc thiết lập đã hoàn tất"
+    IDS_RESTARTSUBTITLE "Giai đoạn đầu của việc thiết lập đã được hoàn thành, khởi động lại máy tính để tiếp tuc với giai đoạn hai"
+    IDS_SUMMARYTITLE "Tóm lược quá trình cài đặt"
+    IDS_SUMMARYSUBTITLE "Liệt kê các thuộc tính đã lựa chọn nhằm kiểm lại trước khi áp dụng vào thiết bị được chọn"
+    IDS_ABORTSETUP "ReactOS vẵn chưa được hoàn tất cài đặt trên máy tính của bạn. Nếu bây giờ bạn hủy trình Thiết lập, lần tới ban sẽ cần phải chạy lại trình Thiết lập để cài đặt ReactOS.Bạn thực sự  muốn bỏ?"
```
Comment (binarymaster) [inline base/setup/reactos/lang/vi-VN,rc]: Text inconsistence:

> `ReactOS.Bạn thực sự  muốn bỏ?`

- No space after a full stop character.
- Two spaces between `sự  muốn`.

**Comment by Dummynosrev** [comment]
Thanks! Little silly questions: How can I adopt/approve the suggested changes? And can you give me the path to the 'reactos.rc' files? Literally an absolute beginner in gitHub.

**Comment by Fraizeraust** [comment]
Since you are implementing the Vietnamese translation for ReactOS GUI Installer, the relative path of `reactos.rc` is in `reactos/base/setup/reactos`. Open the file with a text editor, scroll down until you see `#pragma code_page(65001)`. Below that there's a list of macro directive definitions of translations. The list is in alphabetical order so you must look at the `lang` folder where your translation file stands. Then paste these lines:
```
#ifdef LANGUAGE_LANGUAGECODE_LANGUAGECODE
    #include "lang/translationcodehere.rc"
#endif
```
`LANGUAGECODE` and `translationcodehere` must be replaced accordingly, in your case Vietnamese. As for the requested reviews, just look at them what changes must be done.

**Comment by Dummynosrev** [comment]
Ok. But do I only need to change my github and Reactos account's name my Google account as well. This Email address is the one I use the most frequent to login in to web page and such.

**Comment by Dummynosrev** [comment]
_OR I HAVE TO CHANGE_ my Google account
(Typing mistake)

**Comment by binarymaster** [comment]
> OR I HAVE TO CHANGE my Google account

No, this is not required. Only in your GitHub account.

**Comment by Dummynosrev** [comment]
Ok, thanks!

**Comment by Dummynosrev** [comment]
I add my name, Nguyễn Thanh Sơn, in to my account, it did not replace but just put it aside with the previous name. Did I do something incorrectly?

**Comment by HBelusca** [comment]
OK I've merged all the commits together properly and set the author name as required.
Please update your branch with the git command: `git reset --hard patch-1` .
Only missing stuff @Dummynosrev is whether you want to specify your mail or not.
I'm waiting for a final review before merging.

**Pull request merged**